### DISCUSSION TITLE
What is PEGS4 Material "RMISW" ?

### DISCUSSION THREAD

**Discussion by barghashr** [nrc-cnrc/EGSnrc#1225] [Questions]
Hello, I've been trying to find out if PEGS files have a predefined material for trabecular bone or bone marrow. I couldn't find one, but I did find a material called "RMISW" that has calcium in its components, so I'm curious about what that material is. I have attached a screenshot of the material from the "material.dat" file. 

Thank you.

<img width="763" alt="Screen Shot 2024-11-22 at 20 10 15" src="https://github.com/user-attachments/assets/323998d2-657d-45a9-bedd-f9a388d5e9ad">

**Comment by rtownson** [comment]
There are a few bone variants provided as density correction files, in `HEN_HOUSE/pegs4/density/compounds`. Though I generally recommend that you create your own material to ensure you can verify its correctness, following [these instructions](https://github.com/nrc-cnrc/EGSnrc/wiki/Generating-density-effect-correction-files-with-ESTAR). 

@blakewalters do you know the origin of this material?

**Reply by ojalaj** [reply]
@rtownson Aren't most of the materials based on [ICRU Report 44](https://www.icru.org/report/tissue-substitutes-in-radiation-dosimetry-and-measurement-report-44/)?

**Reply by rtownson** [reply]
Thanks @ojalaj, I believe that is correct.

**Comment by nexustowers** [comment]
Could this be a commercial brand of solid water?

**Comment by crcrewso** [comment]
@nexustowers I think you're right. I found [an article from D I Thwaites](https://iopscience.iop.org/article/10.1088/0031-9155/44/5/001) 

I'm going to guess that it's the brown Gammex (now Sun Nuclear).

**Comment by ftessier** [comment]
@blakewalters, can you add a comment line for each medium in `material.dat`, to spell out what each material is? Or else, please post an issue as a reminder.

**Comment by mainegra** [comment]
**ChatGPT:**
RMI stands for Radiation Measurement Incorporated. It was a company that specialized in the development and production of materials and tools for radiation therapy and dosimetry. Their Solid Water products, designed to simulate the radiological properties of water, are widely used in medical physics for dosimetry and quality assurance in radiation therapy.
Sun Nuclear Corporation, which acquired Gammex in 2014, now manufactures and commercializes Solid Water® HE phantoms, which might be a rebranding of RMI Solid Water.
